### PR TITLE
🌱 Make bootstrap script able to not deploy

### PR DESCRIPTION
**Pull request by MikeSpreitzer**
## Summary
This PR adds an option to the bootstrap script to skip deploying kcp and KubeStellar, to focus only on fetching them.

This PR also updates the commands doc to make all the headings get indexed and remove documentation of the curl-to-bash scripts that we are no longer recommending to users.

This is an alternative to #856 

## Related issue(s)

Fixes #

### CODE REVIEW

**Comment by MikeSpreitzer** [comment]
/cc @francostellari 
/cc @dumb0002 
@ronenkat

**Comment by francostellari** [comment]
@MikeSpreitzer your changes look good to me but I will let @ronenkat have the final say.

**Comment by clubanderson** [comment]
@ronenkat can you review and see if this satisfies your needs?

File: bootstrap/bootstrap-kubestellar.sh [hunk 1]
```diff
@@ -19,6 +19,7 @@
 # This script installs kcp and Kubestellar binaries to a folder of choice
 #
 # Arguments:
+# [--deploy bool] control whether platform deployment is included
```
Comment (ronenkat) [inline bootstrap/bootstrap-kubestellar.sh:22]:
```suggestion
# [--deploy bool] indicate if kubestellar (and kcp) will be deployed
```

**Review by ronenkat — APPROVED** [review]
Looks good.
One change requested

**Comment by clubanderson** [comment]
@ronenkat added you as contributor so that you can lgtm

**Review by clubanderson — COMMENTED** [review]
/approve

**Pull request merged**